AppInstaller/spotless/copyright.txt

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright $YEAR The Android Open Source Project
```

Comment:
$YEAR or 2021? 🙈

Reply:
Spotless will replace this variable by the current year when adding the copyright header on a new file